src/uct/ib/base/ib_iface.h

```diff
@@ -188,11 +188,11 @@ uct_ib_iface_invoke_am(uct_ib_iface_t *iface, uint8_t am_id, void *data,
     void *desc = (char*)ib_desc + iface->config.rx_headroom_offset;
     ucs_status_t status;
 
-    status = uct_iface_invoke_am(&iface->super, am_id, data, length, desc);
+    uct_recv_desc_iface(desc) = &iface->super.super;
```

Comment:
Nope, MM was changed too, see https://github.com/openucx/ucx/pull/1394/files#diff-89d0036f965e77e51861a4c85bf7e906R134
Do you propose add this under `if (status == IN_PROGRESS)` everywhere like it was in MM code?